app/uk/gov/hmrc/plasticpackagingtax/registration/views/liability_weight_page.scala.html

```diff
@@ -1,70 +1,88 @@
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+@import uk.gov.hmrc.govukfrontend.views.html.components.{GovukButton, GovukInput, govukFieldset}
 @import uk.gov.hmrc.govukfrontend.views.html.helpers.formWithCSRF
-@import uk.gov.hmrc.govukfrontend.views.html.components.GovukInput
-@import uk.gov.hmrc.govukfrontend.views.html.components.GovukButton
-@import uk.gov.hmrc.govukfrontend.views.html.components.govukFieldset
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.Title
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.BackButton
 @import uk.gov.hmrc.plasticpackagingtax.registration.controllers.{routes => pptRoutes}
 @import uk.gov.hmrc.plasticpackagingtax.registration.forms.LiabilityWeight
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.paragraphBody
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.sectionHeader
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.errorSummary
-@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.saveButtons
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components.{errorSummary, paragraphBody, saveButtons, sectionHeader}
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.{BackButton, Title}
 
 @this(
-  govukLayout: main_template,
-  paragraphBody: paragraphBody,
-  govukButton: GovukButton,
-  govukInput: GovukInput,
-  sectionHeader: sectionHeader,
-  govukFieldset: govukFieldset,
-  errorSummary: errorSummary,
-  formHelper: formWithCSRF,
-  saveButtons: saveButtons,
+        govukLayout: main_template,
+        paragraphBody: paragraphBody,
+        govukButton: GovukButton,
+        govukInput: GovukInput,
+        sectionHeader: sectionHeader,
+        govukFieldset: govukFieldset,
+        errorSummary: errorSummary,
+        formHelper: formWithCSRF,
+        saveButtons: saveButtons,
+        govukInsetText: GovukInsetText,
 )
 @(form: Form[LiabilityWeight])(implicit request: Request[_], messages: Messages)
 
-@totalWeightField = @{form("totalKg")}
+@totalWeightField = @{
+    form("totalKg")
+}
 
 @errorMessages = @{
     val errors = (form.errors).map(err => messages(err.message, err.key)).mkString("<br>")
     if(errors.nonEmpty) Some(ErrorMessage(content = HtmlContent(errors))) else None
 }
 
+@pptWeightQuestion = {
+    <h1 class="govuk-heading-l">@messages("liabilityWeightPage.question")</h1>
+}
+
+@pptGuidanceLink = {
+    <a id="guidance-link" href="@messages("liabilityWeightPage.guidance.href")" target="_blank">@messages("liabilityWeightPage.guidance.description")</a>
+}
+
+@pptWeightHelp = {
+    <p class="govuk-body">@messages("liabilityWeightPage.info")</p>
+    <p class="govuk-body">@Html(messages("liabilityWeightPage.info2", pptGuidanceLink))</p>
```

Comment:
The Html() call was necessary to ensure that the <a> link was not escaped and rendered rather than being interpreted as a link.